node-services/sharing-service/controllers/reportSharedWithMe.js

```diff
@@ -0,0 +1,36 @@
+const db = require('../dbInstance');
+const { validateEmail, validateUserParameter } = require('./validateParameters');
+
+const dbOperations = {
+  shareWithMe : async (reportObject) => {
```

Comment:
who is me in this context, it might make sense when path is like xyz/user/abc@gmail.com/shareWithMe but not here.

Reply:
Just tried to differentiate b/w two Databases as to track:
1. Reports I have shared with others(sharedWithOthers)
2. Reports which others have shared with me(ReportsSharedWithMe)